src/snapred/backend/dao/state/GroupingMap.py

```diff
@@ -75,8 +82,9 @@ def validate_GroupingMapFile(cls, v):
             for group in groups[mode].copy():
                 fp = Path(groups[mode][group].definition)
                 # Check if path is relative
-                if str(fp) == fp.name:
+                if not os.path.isabs(fp):
```

Comment:
is the original check insufficient?  It seems be based on the idea that if it was not a path it should default to the root grouping folder.

Reply:
The original check was failing.  If the file is `/SNS/SNAP/path/to/some_grouping_definition.hdf`, then the check only passes if the `focusGroup.name` element is `some_grouping_definition`.  However, it's better to allow the file paths and focus group names be unrelated.  So  for instance having a `focusGroup` with `name="All"` and `definition="SNAPFocGroup_All.hdf`, as in the default list, which will here be expanded to point into the state folder

Reply:
I see, sounds reasonable.